src/components/PeoplePage.tsx

```diff
@@ -1,31 +1,155 @@
+import { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
 import { PeopleFilters } from './PeopleFilters';
-import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { SearchOptions } from '../utils/SearchOptions';
+import { SortType } from '../utils/SortType';
+import { SearchParams } from '../utils/searchHelper';
+
+const isIncludeQuery = (
+  personName: Person | undefined,
+  normalizeQuery: string,
+) => (
+  personName?.name.toLocaleLowerCase().trim().includes(normalizeQuery)
+);
+
+const filterPeople = (
```

Comment:
You called the function `filterPeople`, but in the middle of the function you also do `sorting`